runner.yml.dist

```diff
@@ -17,6 +17,7 @@ drupal:
     settings:
       # Allow the test module to be installed.
       extension_discovery_scan_tests: true
+      file_private_path: "sites/default/files/private"
```

Comment:
We would move this out of web root, but for this testing env I don't know if we care.